internal/logger/targets.go

```diff
@@ -131,15 +116,28 @@ func initTargets(cfg Config) (tgts []Target, err error) {
 	return tgts, err
 }
 
-// UpdateTargets swaps targets with newly loaded ones from the cfg
-func UpdateTargets(cfg Config) error {
-	updated, err := initTargets(cfg)
+func initKafkaTargets(cfgMap map[string]kafka.Config) (tgts []Target, err error) {
+	for _, l := range cfgMap {
+		if l.Enabled {
+			t := kafka.New(l)
+			if err = t.Init(); err != nil {
+				return tgts, err
+			}
+			tgts = append(tgts, t)
+		}
+	}
+	return tgts, err
+}
+
+// UpdateHTTPTargets swaps targets with newly loaded ones from the cfg
+func UpdateHTTPTargets(cfg Config) error {
+	updated, err := initHTTPTargets(cfg.HTTP)
 	if err != nil {
 		return err
 	}
 
 	swapMu.Lock()
-	for _, tgt := range targets {
+	for _, tgt := range httpTargets {
```

Comment:
Looks like regression is coming in again @anjalshireesh can you test these things locally? so that we can avoid further regressions on console logging?